spacy/cli/debug_data.py

```diff
@@ -247,6 +255,69 @@ def debug_data(
                     msg.warn(f"No examples for texts WITHOUT new label '{label}'")
                     has_no_neg_warning = True
 
+            with msg.loading("Obtaining span characteristics..."):
+                span_characteristics = _get_span_characteristics(
+                    train_dataset, gold_train_data, spans_key
+                )
+
+            msg.info(f"Span characteristics for spans_key `{spans_key}`")
+            msg.info("SD = Span Distinctiveness, BD = Boundary Distinctiveness")
+            _print_span_characteristics(span_characteristics)
+
+            _span_freqs = _get_spans_length_freq_dist(
+                gold_train_data["spans_length"][spans_key]
+            )
+            _filtered_span_freqs = _filter_spans_length_freq_dist(
+                _span_freqs, threshold=SPAN_LENGTH_THRESHOLD_PERCENTAGE
+            )
+
+            msg.info(
+                f"At least {SPAN_LENGTH_THRESHOLD_PERCENTAGE}% of spans have up to "
```

Comment:
"have up to" seems a little unclear, since it's not obvious that you're necessarily going to count from 1 and it's not that spans contain/have n-grams. I would suggest something more like:

"Over 90% of spans have lengths of 1 -- {NUM}"

"N-gram lengths 1 -- {NUM} At least 90% of spans have lengths 1 -- {NUM}" / "At least 90% of spans are of length 1 -- {NUM}"

Reply:
Noted on this! Updated in https://github.com/explosion/spaCy/pull/10668/commits/9d3c1389f7a4df6fbf5914369b94cf01fe8f5401